openff/bespokefit/executor/services/_settings.py

```diff
@@ -54,6 +54,7 @@ class Settings(BaseSettings):
     BEFLOW_QC_COMPUTE_WORKER = "openff.bespokefit.executor.services.qcgenerator.worker"
     BEFLOW_QC_COMPUTE_WORKER_N_CORES: Union[int, Literal["auto"]] = "auto"
     BEFLOW_QC_COMPUTE_WORKER_MAX_MEM: Union[float, Literal["auto"]] = "auto"
+    BEFLOW_QC_COMPUTE_WORKER_N_TASKS: float = 1
```

Comment:
Maybe for now we should let the spawn method decide this by default? e.g. if the program is one of xtb, ani, or OpenFF for example we should maybe have this be equal to the number of cores, whereas if the program is psi4 this should be 1?

```suggestion
    BEFLOW_QC_COMPUTE_WORKER_N_TASKS: Literal["auto"] = "auto"
```

Reply:
Yes, that could work well, I think we will definitely want to use this with psi4 as well though as the speedup should be significant when the optimization cost is much larger than the time to make the pool. For small molecules splitting the optimisations can slow down the torsiondrive with quick methods like xtb. Also if we go down the auto route we could not expose the variable in the first pass?

Reply:
> psi4 as well though as the speedup should be significant when the optimization cost is much larger than the time to make the pool

Ah good point - maybe something like `n_tasks = n_cores / 8` when using `psi4` as that's how many we normally give workers on QCF?

> For small molecules splitting the optimisations can slow down the torsiondrive with quick methods like xtb. Also if we go down the auto route we could not expose the variable in the first pass?

Oh interesting, that's good to know!

>  Also if we go down the auto route we could not expose the variable in the first pass?

Hmm I think it's still handy to expose as it means we can tweak this easily when trying to optimize different QC methods? Sorry I meant to suggest changing this to `Union[int, Literal["auto"]] = "auto"`!

Reply:
>  Ah good point - maybe something like n_tasks = n_cores / 8 when using psi4 as that's how many we normally give workers on QCF?


Yeah, I think that would be a good idea maybe more generally we could change the setting to control the number of cores per optimization then the number of parallel jobs can be determined from that?